sdk/sdk.go

```diff
@@ -143,6 +152,34 @@ func New(platformEndpoint string, opts ...Option) (*SDK, error) {
 	}, nil
 }
 
+func SanitizePlatformEndpoint(e string) (string, error) {
+	// check if there's a scheme, if not, add https
+	if !regexp.MustCompile(`^https?://`).MatchString(e) {
+		e = "https://" + e
+	}
+
+	if !regexp.MustCompile(`^(https?:\/\/)?(([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}(:\d+)?|(localhost)(:\d+)?)(\/.*)?$`).MatchString(e) {
```

Comment:
```suggestion
	if !regexp.MustCompile(`^(https?:\/\/)?(([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}(:\d+)?|(localhost)(:\d+)?)\/?$`).MatchString(e) {
```
Do we want to allow paths on the platform endpoints?

Reply:
Probably not. Rather tell people they need to run this on the root of the domain.